package.json

```diff
@@ -1,99 +0,0 @@
-{
```

Comment:
woot, thats embarrassing. meant to reset to master, not revert/delete the file! undone.